### ISSUE TITLE
Artisan not working after prestige 

### ISSUE THREAD

**Issue by Estix0** [daleao/modular-overhaul#114]
Both me and host have Agriculturist and Artisan. Host's Agriculturist perk still works after prestige but my Artisan perk stopped working and all wine I produce is normal quality. Version 4.1.4 but don't see any mentions in changelog for 4.1.5.

**Comment by daleao** [comment]
If by Prestige you mean Skill Reset, your chances are based off of Farming level.
So if your Farming level is back to 1, naturally your odds of preserving quality will be very low.

**Comment by Estix0** [comment]
Yes I mean the skill reset at the prestige statue. Both me and host have Farming level 10. Wine produced by the host works as intended (His kegs output different quality wine) while mine is always normal quality.

**Comment by daleao** [comment]
So you mean Artisan was your first choice, and you have leveled all the way back up and chosen Agriculturist second, and throughout all that time your Artisan perk hasn't worked at all?

**Comment by Estix0** [comment]
Exactly, meanwhile host's first perk of choice (Agriculturist) is working properly even after the skill reset.

**Comment by daleao** [comment]
Can you ask the host to share the save please?

**Comment by Estix0** [comment]
Here is the link: https://www.mediafire.com/file/7owxr71upb5wj9n/Stardew_save_and_config.rar/file
I included the current config file as well.

**Comment by RemainingL** [comment]
The same problem happened to me and two of my friends in three different saves. And there's another problem about of lifting the Ruined Blade's Curse--after I finished another four missions, the mission of killing enough monsters which I finished for a huge while becomes unfinished, like the following picture.
![QQ图片20231123203144](https://github.com/daleao/modular-overhaul/assets/151843987/21c05c28-8ce0-498a-a65b-b9819c327f68)

**Comment by daleao** [comment]
@RemainingL please open a separate issue and share your save.

**Comment by daleao** [comment]
@Estix0 
I loaded the host farmer and your farmer in separate game windows.
Simply walked up in your farmhouse and noticed a bunch of Kegs.
Turned on instant Keg production using CJB Cheats Menu and spawned some Iridium Strawberries using CJB Item Spawner.
All of them came back with quality.
So the profession works correctly on my end.
At this point I need you to eliminate the possibility of a mod conflict.

**Comment by Estix0** [comment]
Here is the full list of mods we are using:
[CP] RSV Seasonal Outfits
[CP] Seasonal Cute Characters
[CP] Seasonal Cute Characters SVE
[FS] Ani's colour collection
[FS] Backpacks and Baubles
[FS] Cozy Scarves
[FS] Harry's Short Hair
[FS] More Accessories and Stuff
[FS] Traveller's Attire
AntiSocial NPCs
Console Commands
Content Patches
Custom Companions
Custom NPC Exclusions
Customer nodes
Deep Woods Mod
Error Handler
Expanded Preconditions Utility
Extreme players
Form Type Manager
Fashion Sense
Generic Mod Config Menu
Hoods & Hoodies Pack
Json Assets
Lookup Anything
Mail FrameworkMod
MiniBars
Modular Overhaul
Ridgeside Village
SAAT
Safe Backup
ShopTile Framework
Show Birthdays
SitForStaminaMod
Space Core
Stardew Valley Expanded

**Comment by daleao** [comment]
I can't check that for you.
You have to go through the procedure described in the README.
I.e., start by uninstalling all mods except for this and the requirements, and check if that resolves your issue.

**Comment by daleao** [comment]
Closing due to inactivity. If you do not find the conflict or if you can reproduce the issue with only the minimum requirements, then feel free to revive this thread.
